cypress/e2e/tests/pages/manager/jwt-authentication.spec.ts

```diff
@@ -0,0 +1,148 @@
+
+import JWTAuthenticationPagePo from '@/cypress/e2e/po/pages/cluster-manager/jwt-authentication.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+
+describe('JWT Authentication', { testIsolation: 'off', tags: ['@manager', '@adminUser', '@jenkins'] }, () => {
+  const jwtAuthenticationPage = new JWTAuthenticationPagePo();
+
+  let instance0 = '';
+  let instance1 = '';
+  let removeCluster0 = false;
+  let removeCluster1 = false;
+  const region = 'us-west-1';
+  const namespace = 'fleet-default';
+
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+    cy.createE2EResourceName('rke2cluster0').as('rke2Ec2ClusterName0');
+    cy.createE2EResourceName('rke2cluster1').as('rke2Ec2ClusterName1');
+
+    cy.get<string>('@rke2Ec2ClusterName0').then((name) => {
+      instance0 = name;
+      // create real cluster
+      cy.createAmazonRke2ClusterWithoutMachineConfig({
+        cloudCredentialsAmazon: {
+          workspace: namespace,
+          name,
+          region,
+          accessKey: Cypress.env('awsAccessKey'),
+          secretKey: Cypress.env('awsSecretKey')
+        },
+        rke2ClusterAmazon: {
+          clusterName: name,
+          namespace,
+        }
+      }).then(() => {
+        removeCluster0 = true;
+      });
+    });
+    cy.get<string>('@rke2Ec2ClusterName1').then((name) => {
+      instance1 = name;
+      // create real cluster
+      cy.createAmazonRke2ClusterWithoutMachineConfig({
+        cloudCredentialsAmazon: {
+          workspace: namespace,
+          name,
+          region,
+          accessKey: Cypress.env('awsAccessKey'),
+          secretKey: Cypress.env('awsSecretKey')
+        },
+        rke2ClusterAmazon: {
+          clusterName: name,
+          namespace,
+        }
+      }).then(() => {
+        removeCluster1 = true;
+      });
+    });
+  });
+
+  it('should show the JWT Authentication list page', () => {
+    JWTAuthenticationPagePo.navTo();
+    jwtAuthenticationPage.waitForPage();
+    jwtAuthenticationPage.title().should('be.visible');
+    jwtAuthenticationPage.list().resourceTable().sortableTable().checkVisible();
+    jwtAuthenticationPage.list().resourceTable().sortableTable().checkLoadingIndicatorNotVisible();
+    jwtAuthenticationPage.list().state(instance0).should('contain', 'Disabled');
+    jwtAuthenticationPage.list().state(instance1).should('contain', 'Disabled');
+  });
+  it('should be able to enable JWT Authentication for a cluster', () => {
+    jwtAuthenticationPage.goTo();
```

Comment:
apologies, i should have seen this earlier but this and below tests can use `navTo`s like you have done in the first test. aside from that, this e2e spec lgtm!

  ```
 JWTAuthenticationPagePo.navTo();
 jwtAuthenticationPage.waitForPage();
```

Reply:
Do I need to do that? I don't see a reason to test navigation at every test if we already did it once